Feat: Add schema library section (#103)
UnumID/UnumID.github.io: docs/schema.mdx

Note that each attribute has a `description` and `examples` to help you understand the credential's schema. If you need more detailed information, look at our open source [schema-sdk](https://github.com/UnumID/schema-sdk) [formats](https://github.com/UnumID/schema-sdk/blob/main/src/formats.ts).
:::

## Schema Library

> Next, you can search or select for a schema JSON, see it's properties and attribute, and also copy the schema structure of choice:

Reviewer: ```suggestion
Search for or select a schema, see its properties and attributes, and copy the schema JSON:
```

}

```

> ## Schema Library

Reviewer 2: I would say probably should be featured higher on the page. I would say just under Introduction would be best. Definitely want this to be front and center and the can scroll down for the more wordy explanation. But at the very least this ought to be above Example I think.

Author: Indeed it looks better above example section.
